src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -420,4 +421,58 @@ public void setNotFailingStatuses_SkipsNullValues() {
         // then
         assertThat(configuration.getNotFailingStatuses()).containsExactly(notFailingStatus);
     }
+
+    @Test
+    public void getOutputHandlers_shouldContainFilesystemOutputHandler() {
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        assertThat(configuration.getOutputHandlers()).hasSize(1);
+        assertThat(configuration.getOutputHandlers().get(0)).isInstanceOf(FilesystemOutputHandler.class);
+    }
+
+    @Test
+    public void clearOutputHandlers_shouldRemoveAllOutputHandlersFromOutputHandlersList() {
+        //given
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+        assertThat(configuration.getOutputHandlers()).hasSize(1);
```

Comment:
assertion should not go here
instead you can simple new handler to make sure list is not empty

Reply:
removed assertion